CHANGELOG.md

```diff
@@ -4,15 +4,25 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/).
 
 ## Unpublished
```

Comment:
the "## Unpublished" doesn't go into releases. It's added with the next user-visible change post-release.

Reply:
The first thing on http://keepachangelog.com/ page shows an example where ## [Unreleased] is at the top without anything under. It even says down on the page: "Keep an Unreleased section at the top to track upcoming changes."

Reply:
OK, that's fine. I think the other SDKs may need to change to use this pattern. I'd originally copied the pattern from the OCI SDKs, which are obviously less than perfect in this area.